activerecord/test/cases/load_tree_test.rb

```diff
@@ -0,0 +1,360 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "models/developer"
+require "models/contract"
+require "models/company"
+require "models/computer"
+require "models/mentor"
+require "models/project"
+require "models/ship"
+require "models/ship_part"
+require "models/strict_zine"
+require "models/interest"
+require "models/treasure"
+require "models/pirate"
```

Comment:
I think some of these aren't needed, can you remove the ones that you aren't using in here? At a quick glance, pirate, treasure, and interest aren't used.